src/breadcrumbs/breadcrumbs.scss

```diff
@@ -21,45 +21,94 @@
 
 @mixin iui-breadcrumbs-item {
   display: flex;
+  align-items: center;
   line-height: $iui-component-height;
   height: $iui-component-height;
-  max-width: $iui-3xl * 2;
-  margin: 0 $iui-sm;
 
-  a {
+  // #region Component overrides
+  > a {
     @include iui-anchor;
+    padding: 0 $iui-s;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+
+    &:focus {
+      outline-offset: -1px;
+    }
   }
 
   .iui-button {
     @include iui-button-borderless;
-    margin: 0 (-$iui-s - 1);
+    border: none;
+
+    &-label {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    }
+  }
+
+  > * {
+    max-width: 26ch;
```

Comment:
in date picker its 15, in badge its 20, here 26... what was the reason for this value?

Reply:
I had `max-width: 24ch` set on the anchor & the button label but Mayank thought it made more sense to set it on the anchor and button body instead.  We landed on 26ch only because it did not change the test image results.

Reply:
> We landed on 26ch 

I never suggested 26ch (or any value) fwiw. All II said is that it makes sense to `max-width` on the button/anchor instead of the text.

I had this same question in my other comment: https://github.com/iTwin/iTwinUI/pull/568#discussion_r841635928